src/data/init-moves.ts

```diff
@@ -1568,6 +1568,10 @@ export function initMoves() {
       .attr(GyroBallPowerAttr)
       .bulletMove(),
     new SelfStatusMove(MoveId.HEALING_WISH, ElementalType.PSYCHIC, -1, 10, -1, 0, 4)
+      .condition(
+        (user, _target, _move) =>
+          user.getParty().filter((p) => !p.isFainted()).length > (globalScene.currentBattle.double ? 1 : 0),
```

Comment:
I'm pretty sure getParty does not filter out mons that are invalid for a challenge. So we need to check for that
```suggestion
          user.getParty().filter((p) => !p.isFainted() && p.isAllowedInChallenge()).length > (globalScene.currentBattle.double ? 1 : 0),
```

A comment clarifying the condition could be nice too, it's not super easy to understand at a glance i think

Reply:
There's a helper function for this. Also, this probably needs to filter out active Pokemon:

```suggestion
          user.getParty().filter((p) => !p.isActive(true) && p.isAllowedInBattle()).length > 0,
```